Dashboard landing page reorg (#22296)
DataDog/documentation: content/en/dashboards/_index.md

    {{< nextlink href="/dashboards/configure" >}}Dashboard List: Search, view, or create dashboards and lists{{< /nextlink >}}
    {{< nextlink href="/dashboards/template_variables" >}}Template Variable: Dynamically filter widgets in a dashboard{{< /nextlink >}}
    {{< nextlink href="/service_management/incident_management/datadog_clipboard/" >}}Datadog Clipboard{{< /nextlink >}}
    {{< nextlink href="/api/latest/dashboards" >}}API: Manage dashboards programmatically{{< /nextlink >}}
{{< /whatsnext >}}

{{< whatsnext desc="Graphing features:">}}
    {{< nextlink href="/dashboards/widgets" >}}Widgets: Learn the configuration for different visualizations{{< /nextlink >}}
    {{< nextlink href="/dashboards/querying" >}}Querying: See the formatting options for graph queries{{< /nextlink >}}
    {{< nextlink href="/dashboards/functions" >}}Functions: Modify metric queries and resulting graphs{{< /nextlink >}}
    {{< nextlink href="/dashboards/change_overlays" >}}Overlays: Automatically overlay change events on graphs{{< /nextlink >}}
{{< /whatsnext >}}

## Get started

> {{< whatsnext desc="See the following resources to get started with Dashboards:" >}}

Reviewer: ```suggestion
{{< whatsnext desc="See the following resources:" >}}
```
Just because the repetition here with the first item in the list feels a little odd, especially when the heading is also "Get started".

>    {{< nextlink href="/getting_started/dashboards/" >}}Getting started with Dashboards{{< /nextlink >}}

Reviewer: ```suggestion
   {{< nextlink href="/getting_started/dashboards/" >}}Getting Started with Dashboards{{< /nextlink >}}
```

   {{< nextlink href="https://learn.datadoghq.com/courses/intro-dashboards" >}}Learning Course: Introduction to Dashboards{{< /nextlink >}}
   {{< nextlink href="https://learn.datadoghq.com/courses/building-better-dashboards" >}}Learning Course: Building Better Dashboards{{< /nextlink >}}
{{< /whatsnext >}}

To create a dashboard, click **+New Dashboard** on the [Dashboard List][4] page or **New Dashboard** from the navigation menu. Enter a dashboard name and choose a layout option.

{{< img src="dashboards/create-dashboard.png" alt="Adding a new dashboard" style="width:70%;">}}

Dashboards 

> : A grid-based layout, which can include a variety of objects such as images, graphs, and logs. They are commonly used as status boards or storytelling views which update in realtime, and can represent fixed points in the past. They have a maximum width of 12 grid squares and also work well for debugging.

Reviewer: ```suggestion
: A grid-based layout, which can include a variety of objects such as images, graphs, and logs. They are commonly used as status boards or storytelling views which update in real time, and can represent fixed points in the past. They have a maximum width of 12 grid squares and also work well for debugging.
```

Timeboards
: Automatic layouts that represent a single point in time—either fixed or real-time—across the entire dashboard. They are commonly used for troubleshooting, correlation, and general data exploration.

Screenboards

> : Dashboards with free-form layouts which can include a variety of objects such as images, graphs, and logs. They are commonly used as status boards or storytelling views that update in real-time or represent fixed points in the past.

Reviewer: ```suggestion
: Dashboards with free-form layouts which can include a variety of objects such as images, graphs, and logs. They are commonly used as status boards or storytelling views that update in real time or represent fixed points in the past.
```

| 6 months     | 1 hour       |
| 1 year       | 1 hour       |

## View dashboards on mobile devices

> View your dashboards in a mobile-friendly format with the Datadog Mobile App, available on the [Apple App Store][2] and [Google Play Store][3]. The Mobile App comes equipped with mobile home screen widgets which allow you to monitor service health and infrastructure without opening the mobile app.

Reviewer: ```suggestion
View your dashboards in a mobile-friendly format with the Datadog Mobile App, available on the [Apple App Store][2] and [Google Play Store][3]. The Mobile App comes equipped with mobile home screen widgets that allow you to monitor service health and infrastructure without opening the mobile app.
```
